circle.yml

```diff
@@ -4,6 +4,8 @@ general:
       - gh-pages
 
 machine:
+  pre:
+    - curl -sSL https://s3.amazonaws.com/circle-downloads/install-circleci-docker.sh | bash -s -- 1.10.0
```

Comment:
This line is only necessary until CircleCI switches to a newer version of Docker, right? Maybe add a comment to remove it when that happens (not sure how much this step is slowing down the whole build).